### PR TITLE
(example/with-antd-mobile): Fix the example

### PR DESCRIPTION
**Pull request by akellbl4**
### Documentation / Examples

Fixed one of the items from #25854

- Removes `next/with-css`
- Excludes import CSS inside `antd-mobile` in any case (for frontend and backend)
- Fixes regexp for `null-loader`
- Import `andt-mobile` styles in `_app.js`
- Move style from `Head` because it caused an error
- Use `Router.push` instead of using `withRouter` HOC

---

- [x] Add the StackBlitz button in README.md
- [x] Make sure the linting passes

[Live demo on StackBlitz](https://stackblitz.com/github/akellbl4/next.js/tree/fix-examples-with-ant-design-mobile/examples/with-ant-design-mobile)

### CODE REVIEW

**Comment by d3lm** [comment]
I think this is also missing some peer dependencies. Can you also fix those in this PR? Here's a link to the [live preview](https://stackblitz.com/github/akellbl4/next.js/tree/fix/example-with-i18n-next-intl-missing-other/examples/with-ant-design-mobile) from your fork.

**Comment by d3lm** [comment]
And once the error is fixed, can you also add the StackBlitz button to the `README.md`? Here's the snippet:
```md
## Preview

Preview the example live on [StackBlitz](http://stackblitz.com/):

[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/vercel/next.js/tree/canary/examples/with-ant-design-mobile)
```

File: examples/with-ant-design-mobile/README.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 This example features how you use [antd-mobile](https://github.com/ant-design/ant-design-mobile) (Ant Design Mobile FrontEnd Framework) with Next.js.
 
+## Preview
+
+Preview the example live on [StackBlitz](http://stackblitz.com/):
+
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/vercel/next.js/tree/canary/examples/with-ant-design-mobile)
```
Comment (d3lm) [inline examples/with-ant-design-mobile/README.md:9]: Note that we should only merge this if it works on StackBlitz, that is we fix the missing peer dependency issue.

**Comment by akellbl4** [comment]
@d3lm as I can see you created the live example, not with this branch.
Here is the right [live example](https://stackblitz.com/github/akellbl4/next.js/tree/fix-examples-with-ant-design-mobile/examples/with-ant-design-mobile)

**Comment by d3lm** [comment]
Oh, my bad! Sorry about that. Thanks for linking the correct one!

**Comment by akellbl4** [comment]
This example has been removed by Tim in [this PR](https://github.com/vercel/next.js/pull/26075). I've restored it because it had been fixed :)

**Comment by balazsorban44** [comment]
Hi, this PR seems to be a bit out-of-date with our [example guidelines](https://github.com/vercel/next.js/blob/canary/contributing/examples/adding-examples.md), so I am going to close it. If you have the intention to bring it up-to-date, please do! Thanks for the work either way! :pray: :green_heart:

**Pull request closed without merging**